demosplan/DemosPlanCoreBundle/Command/ContainerInitCommand.php

```diff
@@ -50,17 +76,127 @@ public function configure(): void
      * @throws Exception
      */
     public function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $exitCode = $this->initializeDatabase($output);
+        if (self::SUCCESS !== $exitCode) {
+            return $exitCode;
+        }
+
+        $exitCode = $this->initializeCustomer($input, $output);
+        if (self::SUCCESS !== $exitCode) {
+            return $exitCode;
+        }
+
+        return self::SUCCESS;
+    }
+
+    protected function initializeDatabase(OutputInterface $output): int
     {
         $connection = $this->entityManager->getConnection();
         try {
             $connection->getDatabase();
         } catch (ConnectionException $throwable) {
-            // create database, if it does not exist yet
-            Batch::create($this->getApplication(), $output)
-                ->add('dplan:db:init --with-fixtures=ProdData --create-database')
-                ->run();
+            try {
+                // create database, if it does not exist yet
+                Batch::create($this->getApplication(), $output)
+                    ->add('dplan:db:init --with-fixtures=ProdData --create-database')
+                    ->run();
+            } catch (Exception $exception) {
+                $output->writeln(
+                    "Something went wrong during database initialization: {$exception->getMessage()}",
+                    OutputInterface::VERBOSITY_NORMAL
+                );
+
+                return self::FAILURE;
+            }
+        }
+
+        return self::SUCCESS;
+    }
+
+    private function initializeCustomer(InputInterface $input, OutputInterface $output): int
+    {
+        try {
+            $config = $this->getCustomerConfig($input);
+            if (null === $config) {
+                return Command::SUCCESS;
+            }
+
+            $customerName = $config[CustomerConfiguration::NAME];
+            $customerSubdomain = $config[CustomerConfiguration::SUBDOMAIN];
+            $userLogin = $config[CustomerConfiguration::USER_LOGIN] ?? null;
+
+            /** @var Customer $customer */
+            /** @var User $user */
+            [$customer, $user] = $this->transactionService->executeAndFlushInTransaction(
+                function () use (
+                    $customerName,
+                    $customerSubdomain,
+                    $userLogin
+                ): array {
+                    $customer = $this->customerService->createCustomer($customerName, $customerSubdomain);
+                    if (null === $userLogin) {
+                        return [$customer, null];
+                    }
+
+                    $user = $this->userRepository->getFirstUserByCaseInsensitiveLogin($userLogin);
+                    if (null === $user) {
+                        $user = $this->userService->createMasterUserForCustomer(
+                            $userLogin,
+                            $customer
+                        );
+                    }
+
```

Comment:
does this user not need an organisation as well?

Reply:
As far I was told this user is only needed for technical reasons (if at all) and thus should not need an organisation. The plan is to test it without creating the user (this is why the `userLogin` field in the configuration is optional). The  code for the creation of the user is only left in in case it does not work otherwise and can be removed if it is not needed.